leetcode/string_1.rs

```diff
@@ -0,0 +1,19 @@
+//Defanging IP Address
+pub fn defang_i_paddr(address: String) -> String {
+    address.replace(".", "[.]")
+}
+
+//Jewels and Stones
+pub fn num_jewels_in_stones(jewels: String, stones: String) -> i32 {
+    stones
+        .chars()
+        .filter(|&stones| jewels.contains(stones))
+        .count() as i32
+}
+
+//Maximum words in sentence
+pub fn most_words_found(sentences: Vec<String>) -> i32 {
```

Comment:
This function panics if `sentences` is an empty vector. Rather, it would be better to return 0.